components/card/themes/spectrum-two.css

```diff
@@ -67,7 +67,7 @@
 		--spectrum-card-selected-background-opacity: 0.1; /* table-selected-row-background-opacity does not exist in tokens yet */
 
 		--spectrum-card-preview-border-width-selected: var(--spectrum-border-width-100);
-		--spectrum-card-preview-background-color: var(--spectrum-background-base-color);
+		--spectrum-card-preview-background-color: var(--spectrum-gray-200);
```

Comment:
If we're changing this for spectrum-two, shouldn't `spectrum.css` set this back to the Spectrum 1 value of `--spectrum-background-base-color`?

Reply:
I see this is updated now, thanks.